src/uwtools/drivers/forecast.py

```diff
@@ -282,12 +281,13 @@ def _run_via_local_execution(self) -> Tuple[bool, List[str]]:
         :return: A tuple containing a boolean of the success status of the FV3 run and a list of
             strings that make up the full command line.
         """
+        run_directory = self.prepare_directories()
         pre_run = self._mpi_env_variables(" ")
         full_cmd = f"{pre_run} {self.run_cmd()}"
         command_lines = ["Command:", *full_cmd.split("\n")]
         if self._dry_run:
             return True, command_lines
-        result = execute(cmd=full_cmd)
+        result = execute(cmd=full_cmd, cwd=run_directory, log_output=True)
```

Comment:
Ah good fix - I would have thought it would work implicitly but there's no replacement for direct testing!